react/src/paybutton-config.json

```diff
@@ -1,4 +1,9 @@
 {
   "wsBaseUrl": "https://socket.paybutton.org",
-  "apiBaseUrl": "https://paybutton.org"
+  "apiBaseUrl": "https://paybutton.org",
+  "defaultCurrency": "xec",
+  "defaultWallets": {
```

Comment:
This should be called `Addresses` instead of `Wallets`, so considering the previous comment `defaultStorybookAddresses`